CATesting/config.yaml

```diff
@@ -0,0 +1,119 @@
+# Constants
```

Comment:
It's a good idea to follow the convention of the repo when naming files and folders. It's also important to keep in mind that different setups will use case sensitive or case insensitive names. 

This folder name should also be placed somewhere in the repo that is well organized with respect to the rest of the code, this isn't a top level concept for AKT.

I'd probably do 
/eks/cluster-loader/configs/cluster-autoscaler

Reply:
Got it. I'll cancel this one and do a different one

Reply:
Got it.